pkgs/top-level/all-packages.nix

```diff
@@ -9354,6 +9354,8 @@ with pkgs;
 
   incron = callPackage ../tools/system/incron { };
 
+  indexed_bzip2 = with python3Packages; toPythonApplication indexed_bzip2;
```

Comment:
Seems like the `_` -> `-` didn't get applied here. Perhaps this is the cause of the eval failure?

Reply:
Fixed. Thank you for spotting this.